### PR TITLE
Add Aspire Stage Three announcement and banner

### PR DESCRIPTION
**Pull request by JBHyperion**
### Description

This PR adds a news announcement of the forthcoming Aspire: Stage Three beatmapping contest for osu!catch, along with relevant banner graphic.

### Status

Pending Review

### Changelog

21/09/17 - Added banner and links to contest page / .osz

### CODE REVIEW

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+With the osu! and osu!taiko communities flexing their creative muscles in the past two Aspire contests, the time is ripe for the fruit-catchers to take the spotlight, to the sound our very own Featured Artist, **OISHII**!
+
+![](/wiki/shared/news/banners/osu_catch.png)
+
+Casting down the Ranking Criteria and boldly venturing forth into a world where (almost) anything goes, Aspire is the premier "no-rules" official beatmapping contest.
+
+On two occasions already this year, we have seen some of the community's best and brightest mappers step up to this challenge. Between the somber tones of Helblinde's **The Solace of Oblivion** in osu! and the intensity of Function Phantom's **Algebra** in osu!taiko, Aspire has showcased some truly mind-blowing performances that have completely changed the landscape of modern mapping.
+
+Now it is time for the osu!catch community to join the fray, and as countless fruits rain down from above, the sky is no longer the limit! In an effort to give our extrodinarily talented Featured Artists the exposure they deserve, the osu!catch community will begin their contest with a track and artist that has excited many mappers since their unveiling:
```
Comment (mangomizer) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: extrodinarily is not a word, I'm guessing brain went autopilot here.
change to extraordinarily OWTTE

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+Similar to the previous two Aspire Stages, **this contest will be judged by a combination of double-blind community voting and the opinions of an expert judging panel weighted equally against each other.** Make sure you appeal to our judges and give the community something to take notice of if you want to claim the prize! Remember that this is not an ordinary beatmapping contest, so get creative and  use everything and anything at your disposal to stand out!
+
+After shortlisting the best and most innovative entries, we will aim to provide a livestream showcase alongside expert commentary from renowned and established osu!catch community members as well.
+
+After both community voting and feedback from our expert judging panel, the top three placing maps shall receive:
+
+* **1st place**: 6 months of osu!supporter, a unique Aspire 2017: Stage Three profile badge, an osu!news showcase featuring their victorious entry and an *Elite Mapper: Aspirant* forum title (or special incrementation if they already possess one). The map shall also be forever enshrined as an Aspire class Approval map with fast-track help from our wonderful Beatmap Nominators if required.
```
Comment (mangomizer) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: winning is fine in place of victorious?

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+On two occasions already this year, we have seen some of the community's best and brightest mappers step up to this challenge. Between the somber tones of Helblinde's **The Solace of Oblivion** in osu! and the intensity of Function Phantom's **Algebra** in osu!taiko, Aspire has showcased some truly mind-blowing performances that have completely changed the landscape of modern mapping.
+
+Now it is time for the osu!catch community to join the fray, and as countless fruits rain down from above, the sky is no longer the limit! In an effort to give our extrodinarily talented Featured Artists the exposure they deserve, the osu!catch community will begin their contest with a track and artist that has excited many mappers since their unveiling:
+
+Courtesy of **OISHII**, take a trip down the *Onigiri Freeway*!
+
+<iframe width="700" height="394" src="https://www.youtube.com/watch?v=AKZd1zMxMlg?rel=0" frameborder="0" allowfullscreen></iframe>
+
+A fast and furious sugar-rush of synthesizers and sweetness, *Onigiri Freeway* presents a cheerful yet high-octane challenge for the  osu!catch community to tackle.
```
Comment (Ephemeralis) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: thar be an extra space hiding before `osu!catch` here

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* The map itself must be able to be passed (note: NOT fc'd) by a human player without using difficulty reducing mods of any kind.
+* The drain time of your entry must be at least 70% of the song's total length.
+* **If your entry has no additional assets (hitsounds, backgrounds, etc.), it MUST BE SUBMITTED AS A .osu file.** 
+* **If your entry HAS additional assets such as storyboard assets, hitsounds, etc., it MUST BE SUBMITTED AS A .osz with a size NO LARGER THAN 20mb.** Accounting for the size of the track, you have roughly **16mb** of asset space available for use. Extensions may be granted upon request via forum PM to [Ephemeral](https://osu.ppy.sh/users/102335).
+
+
+Similar to the previous two Aspire Stages, **this contest will be judged by a combination of double-blind community voting and the opinions of an expert judging panel weighted equally against each other.** Make sure you appeal to our judges and give the community something to take notice of if you want to claim the prize! Remember that this is not an ordinary beatmapping contest, so get creative and  use everything and anything at your disposal to stand out!
```
Comment (Ephemeralis) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: another double space at `use`

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+![](/wiki/shared/news/banners/osu_catch.png)
+
+Casting down the Ranking Criteria and boldly venturing forth into a world where (almost) anything goes, Aspire is the premier "no-rules" official beatmapping contest.
+
+On two occasions already this year, we have seen some of the community's best and brightest mappers step up to this challenge. Between the somber tones of Helblinde's **The Solace of Oblivion** in osu! and the intensity of Function Phantom's **Algebra** in osu!taiko, Aspire has showcased some truly mind-blowing performances that have completely changed the landscape of modern mapping.
+
+Now it is time for the osu!catch community to join the fray, and as countless fruits rain down from above, the sky is no longer the limit! In an effort to give our extrodinarily talented Featured Artists the exposure they deserve, the osu!catch community will begin their contest with a track and artist that has excited many mappers since their unveiling:
+
+Courtesy of **OISHII**, take a trip down the *Onigiri Freeway*!
```
Comment (Ephemeralis) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: link to featured artist page for OISHII

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Aspire Stage Three: osu!catch Begins"
+date: 2017-09-22 00:00:00 +0100
```
Comment (Ephemeralis) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: change time to 22:00:00

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+The rules are as follows:
+
+* This contest is for the **osu!catch** mode only.
+* The osu!catch and General Ranking Criteria **DO NOT APPLY** to this contest in any capacity whatsoever.
+* You may submit **ONE** difficulty to the contest **OF YOUR OWN CREATION**, be it collaborative or otherwise.
+* The map you submit may be of any difficulty tier you wish, though Aspire contest entries are typically geared towards Expert (5.25+ star) tier and above.
+* The map itself must be able to be passed (note: NOT fc'd) by a human player without using difficulty reducing mods of any kind.
+* The drain time of your entry must be at least 70% of the song's total length.
+* **If your entry has no additional assets (hitsounds, backgrounds, etc.), it MUST BE SUBMITTED AS A .osu file.** 
+* **If your entry HAS additional assets such as storyboard assets, hitsounds, etc., it MUST BE SUBMITTED AS A .osz with a size NO LARGER THAN 20mb.** Accounting for the size of the track, you have roughly **16mb** of asset space available for use. Extensions may be granted upon request via forum PM to [Ephemeral](https://osu.ppy.sh/users/102335).
```
Comment (mangomizer) [inline news/2017-09-22-aspire-stage-three-osucatch-begins.md]: MB not mb?

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 6 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: done

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 1 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: no I just can't type ok

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 5 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: done

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 3 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: killed it

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 7 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: I got confused like what does Magic Bomb have to do with this but then I got it, fixed

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 4 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: anotha one

File: news/2017-09-22-aspire-stage-three-osucatch-begins.md (hunk 2 above)
Comment (JBHyperion) [reply news/2017-09-22-aspire-stage-three-osucatch-begins.md]: boring'ed it ):

**Pull request merged**